src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -504,6 +592,8 @@ void NotationViewInputController::mouseReleaseEvent(QMouseEvent*)
 
     if (viewInteraction()->isDragStarted()) {
         viewInteraction()->endDrag();
+    } else if (m_pressedOnSelected && viewInteraction()->selection()->element()->isTextBase()) {
+        viewInteraction()->startEditText(viewInteraction()->selection()->element(), PointF(m_view->toLogical(event->pos())));
```

Comment:
as per MU3, when clicking on an already selected text element, start text edit mode - but it only occurs on button *release* (whereas the selection happens on press).